### PR TITLE
XS✔ ◾ Added Drawbacks to GTM rule

### PR DESCRIPTION
**Pull request by Harry-Ross**

### CODE REVIEW

File: rules/use-google-tag-manager/rule.md [hunk 1]
```diff
@@ -44,4 +44,14 @@ Data and analytics result has recorded by Google Analytics on new SSW.Website. U
...
\ No newline at end of file
+Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
+
+### Drawbacks
+While Google Tag Manager is great for non-tech people to easily embed tracking code, you must be careful to not introduce functionality to the website that may interfere with existing systems. 
+
+Some things to consider when adding GTM scripts to a container:
+
+* **Performance** - some scripts may have a large performance impact, and result in the slowing of the site. 
+* **Unexpected side effects** - scripts may interfere with existing systems on the site
```
Comment (AttackOnMorty) [inline rules/use-google-tag-manager/rule.md:55]: This point is kind of vague. Could you elaborate more?

File: rules/use-google-tag-manager/rule.md [hunk 2]
```diff
@@ -44,4 +44,14 @@ Data and analytics result has recorded by Google Analytics on new SSW.Website. U
...
-Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
\ No newline at end of file
+Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
+
+### Drawbacks
+While Google Tag Manager is great for non-tech people to easily embed tracking code, you must be careful to not introduce functionality to the website that may interfere with existing systems. 
+
+Some things to consider when adding GTM scripts to a container:
+
+* **Performance** - some scripts may have a large performance impact, and result in the slowing of the site.
```
Comment (camillars) [inline rules/use-google-tag-manager/rule.md:54]: any examples to mention here?

File: rules/use-google-tag-manager/rule.md [hunk 3]
```diff
@@ -44,4 +44,14 @@ Data and analytics result has recorded by Google Analytics on new SSW.Website. U
...
+Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
+
+### Drawbacks
+While Google Tag Manager is great for non-tech people to easily embed tracking code, you must be careful to not introduce functionality to the website that may interfere with existing systems. 
+
+Some things to consider when adding GTM scripts to a container:
+
+* **Performance** - some scripts may have a large performance impact, and result in the slowing of the site. 
+* **Unexpected side effects** - scripts may interfere with existing systems on the site
+* **Duplication of code** - some functionality may already be implemented by developers within the site, which may cause a duplication of logic.
```
Comment (camillars) [inline rules/use-google-tag-manager/rule.md:56]: how can devs stop this from happening?

File: rules/use-google-tag-manager/rule.md [hunk 4]
```diff
@@ -44,4 +44,14 @@ Data and analytics result has recorded by Google Analytics on new SSW.Website. U
...
+* **Disconnected functionality** - some functionality may require developer modifications to ensure that it works correctly, and these customizations may enhance user experience.
```
Comment (camillars) [inline rules/use-google-tag-manager/rule.md:57]: again, any examples? this is good to mention but at SSW we do have access to devs, so maybe we need an example.

File: rules/use-google-tag-manager/rule.md (hunk 2 above)
Comment (bradystroud) [inline rules/use-google-tag-manager/rule.md:54]:
```suggestion
* **Performance** - some scripts may have a large performance impact, and result in the slowing of the site. The more scripts you add, the slower the site becomes
```
checked by @camillars @Harry-Ross @AttackOnMorty

File: rules/use-google-tag-manager/rule.md (hunk 1 above)
Comment (bradystroud) [inline rules/use-google-tag-manager/rule.md:55]:
```suggestion
```
checked by @camillars @Harry-Ross @AttackOnMorty

File: rules/use-google-tag-manager/rule.md (hunk 4 above)
Comment (bradystroud) [inline rules/use-google-tag-manager/rule.md:57]:
```suggestion
* **Disconnected functionality** - some functionality may require developer modifications in the code that are not possible in GTM to ensure that it works correctly. (e.g. Application Insights)
```

File: rules/use-google-tag-manager/rule.md (hunk 3 above)
Comment (bradystroud) [inline rules/use-google-tag-manager/rule.md:56]:
```suggestion
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by camillars — APPROVED** [review]

**Review by AttackOnMorty — APPROVED** [review]
LGTM

**Review by bradystroud — APPROVED** [review]

**Pull request merged**